cmd/main.go

```diff
@@ -138,6 +138,10 @@ func (o *Options) InitDefaults() {
 	o.ExtraPromptPaths = []string{}
 	o.TracePath = filepath.Join(os.TempDir(), "kubectl-ai-trace.txt")
 	o.RemoveWorkDir = false
+	o.ToolConfigPath = []string{
+		"{CONFIG}/kubectl-ai/tools.yaml",
+		"{HOME}/.config/kubectl-ai/tools.yaml",
```

Comment:
Pl. use the path separator from the os or file package to ensure we support windows as well.

Reply:
done